Dockerfile

```diff
@@ -3,11 +3,12 @@
 # docker run -it lazygit:latest
 
 FROM golang:alpine
+WORKDIR /go/src/github.com/jesseduffield/lazygit/
+COPY ./ .
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o lazygit .
```

Comment:
From what I've read, `-a -installsuffix` allows you to have a fully static binary (C libraries included. The installsuffix is to avoid conflicts if using netgo. Probably not needed here, but I guess the Docker folks just wanted to be explicit).